InterviewFlashCardsTests/InterviewFlashCardsTests.m

```diff
@@ -22,7 +13,6 @@ @implementation InterviewFlashCardsTests
 
 - (void)setUp {
     [super setUp];
-    self.qman = [IFCQueryManager new];
```

Comment:
I didn't delete it since it's part of the testing code, but maybe later on if the project is converted to 100% swift?